javascriptv3/example_code/lambda/lambda_create_function/src/LambdaApp/index.html

```diff
@@ -0,0 +1,29 @@
+<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3. This example is in the 'AWS SDK for JavaScript v3 Developer Guide' at
+https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/lambda-create-table-example.html.
+
+Purpose:
+index.html is the front-end HTML code for a tutorial demonstrating how to build and deploy an app that transcribes and displays
+voice recordings for authenticated users. To run the full tutorial, see
+https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/lambda-create-table-example.html.
+
+*/
```

Comment:
I may have missed something, but I don't see an opening /* for this.

Reply:
As it is HTML, I used the <!-- --> commenting. The '*/' was rogue, and now deleted.